portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeAccountNav.vm

```diff
@@ -82,9 +84,15 @@ For opening help sidebar
     </li>
   #end
     <li>
-      <a class="sak-sysInd-account nav-link text-white" data-bs-toggle="offcanvas" href="#sakai-accountPanel" role="button" aria-controls="sakai-accountPanel" title="Account">
-        <img class="sakai-accountProfileImage rounded-circle" width="50" src="/direct/profile/${loginUserId}/image/thumb" alt="Profile image">
-      </a>
-      <!--button class="btn only-icon" data-bs-toggle="modal" data-bs-target="#profile-image-upload">balls</button-->
+      <button class="btn icon-button sak-sysInd-account"
+          type="button"
+          data-bs-toggle="offcanvas"
+          data-bs-target="#sakai-account-panel"
+          aria-controls="sakai-account-panel"
+          title="${rloader.mainnav_btn_account}">
+        <img class="sakai-accountProfileImage rounded-circle"
+            src="/direct/profile/${loginUserId}/image/thumb"
+            alt="${rloader.sit_profile_image_alt}">
```

Comment:
It seems beneficial to implement a fallback image in case the primary image fails to load. Could we consider adding something like this? `onerror="this.onerror=null; this.src='/path/to/default/image.png'">`